cmd/bridge-auth-config-update.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/0chain/gosdk/zcncore"
+	"github.com/spf13/cobra"
+	"log"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+var updateAuthorizerConfigCmd = &cobra.Command{
+	Use:   "bridge-auth-config-update",
+	Short: "Update ZCNSC authorizer settings by ID",
+	Long:  `Update ZCNSC authorizer settings by ID.`,
+	Args:  cobra.MinimumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		const (
+			IDFlag  = "id"
+			FeeFlag = "fee"
+		)
+
+		var (
+			flags = cmd.Flags()
+			ID    string
+			Fee   string
+			err   error
+		)
+
+		if flags.Changed(IDFlag) {
+			if ID, err = flags.GetString(IDFlag); err != nil {
+				log.Fatalf("error in '%s' flag: %v", IDFlag, err)
+			}
+		}
+
+		if flags.Changed(FeeFlag) {
```

Comment:
Return error if "fee" is not provided

Reply:
flag is marked as required, so error is return is not required

```go
cmd.PersistentFlags().String("fee", "", "fee")
cmd.MarkFlagRequired("fee")
```